README.rst

```diff
@@ -10,15 +10,15 @@ omf
     :alt: Documentation
 
 .. image:: https://img.shields.io/badge/license-MIT-blue.svg
-    :target: https://github.com/MiraGeoscience/omf/blob/develop/LICENSE
+    :target: https://github.com/MiraGeoscience/omf/blob/main/LICENSE
```

Comment:
hmm but we are not merging to main right now, pretty sure we sent everything to develop

Reply:
LICENSE has not changed between main and develop.
My point is that once the changes get to main, the main branch should not reference a LICENSE in develop branch.

Reply:
Got it